teamcity: failed tests on master: test/TestLogic, test/TestLogic/distsql, test/TestLogic/distsql/sequences, test/TestLogic, test/TestLogic/distsql, test/TestLogic/distsql/sequences

The following tests appear to have failed:

[#451060](https://teamcity.cockroachdb.com/viewLog.html?buildId=451060):


```
--- FAIL: test/TestLogic (89.490s)
test_log_scope.go:78: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic502227781
test_log_scope.go:61: use -show-logs to present logs inline
--- FAIL: test/TestLogic/distsql (0.530s)
None
--- FAIL: test/TestLogic/distsql/sequences (1.400s)
logic_test.go:2054: 
	testdata/logic_test/sequences:367: SELECT * FROM txn_test
	expected:
	    1 foo
	    3 baz
	but found (query options: "") :
	    3  baz  
	    1  foo  
	
--- FAIL: test/TestLogic (89.490s)
test_log_scope.go:78: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic502227781
test_log_scope.go:61: use -show-logs to present logs inline
--- FAIL: test/TestLogic/distsql (0.530s)
None
--- FAIL: test/TestLogic/distsql/sequences (1.400s)
logic_test.go:2054: 
	testdata/logic_test/sequences:367: SELECT * FROM txn_test
	expected:
	    1 foo
	    3 baz
	but found (query options: "") :
	    3  baz  
	    1  foo  
	

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/logic_test/sequences
+++ b/pkg/sql/logictest/testdata/logic_test/sequences
@@ -364,7 +364,7 @@ ROLLBACK
 statement ok
 INSERT INTO txn_test (something) VALUES ('baz')
 
-query IT
+query IT rowsort
 SELECT * FROM txn_test
 ----
 1 foo

